client.lua

```diff
@@ -1071,19 +1073,33 @@ local function SpawnPed(data)
 		if nextnumber <= 0 then nextnumber = 1 end
 
 		Config.Peds[nextnumber] = data
+		createdPeds[#createdPeds + 1] = data.currentpednumber
 	end
+	return createdPeds
 end
 
 exports('SpawnPed', SpawnPed)
 
 local function RemovePed(peds)
+
+	local function removePedByNumber(pedNumber)
```

Comment:
It looks like this could be a typo of placing a function declaration inside of a function declaration.